### ISSUE TITLE
custom model logger output have not eval and rollout

### ISSUE THREAD

**Issue by fishingcatgo** [hill-a/stable-baselines#1204]
-----------------------------------------
| time/                   |             |
|    fps                  | 26          |
|    iterations           | 7           |
|    time_elapsed         | 136         |
|    total_timesteps      | 3584        |
| train/                  |             |
|    approx_kl            | 0.010918302 |
|    clip_fraction        | 0.0658      |
|    clip_range           | 0.2         |
|    entropy_loss         | -14.2       |
|    explained_variance   | 0.00961     |
|    learning_rate        | 0.0002      |
|    loss                 | 20.1        |
|    n_updates            | 18          |
|    policy_gradient_loss | -0.00807    |
|    std                  | 1           |
|    value_loss           | 36.8        |
-----------------------------------------
